src/Microsoft.ManagedZLib/benchmarks/Microsoft.ManagedZLib.Benchmarks.csproj

```diff
@@ -7,9 +7,14 @@
     <Nullable>enable</Nullable>
```

Comment:
Can you remove line 6? `<ImplicitUsings>enable</ImplicitUsings>`

I'm a bit paranoid because I don't want System.IO.Compression to get autoresolved. Better to indicate it explicilty where needed.

Reply:
Note that you'll get build errors after removing that line, both in CompressedFile.cs and in ManagedZLibBenchmarks.cs.